flake.lock

```diff
@@ -0,0 +1,587 @@
+      },
+      "original": {
+        "type": "tarball",
+        "url": "https://github.com/NixOS/nixpkgs/archive/356624c12086a18f2ea2825fed34523d60ccc4e3.tar.gz"
+      }
+    },
+    "nixpkgs_2": {
+      "locked": {
+        "lastModified": 1717432640,
+        "narHash": "sha256-+f9c4/ZX5MWDOuB1rKoWj+lBNm0z0rs4CK47HBLxy1o=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "88269ab3044128b7c2f4c7d68448b2fb50456870",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "release-24.05",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "nixpkgs_3": {
+      "locked": {
+        "lastModified": 1716977621,
+        "narHash": "sha256-Q1UQzYcMJH4RscmpTkjlgqQDX5yi1tZL0O345Ri6vXQ=",
+        "owner": "cachix",
+        "repo": "devenv-nixpkgs",
+        "rev": "4267e705586473d3e5c8d50299e71503f16a6fb6",
+        "type": "github"
+      },
+      "original": {
+        "owner": "cachix",
+        "ref": "rolling",
+        "repo": "devenv-nixpkgs",
+        "type": "github"
+      }
+    },
+    "nixpkgs_4": {
+      "locked": {
+        "lastModified": 1734126203,
+        "narHash": "sha256-0XovF7BYP50rTD2v4r55tR5MuBLet7q4xIz6Rgh3BBU=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "71a6392e367b08525ee710a93af2e80083b5b3e2",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixpkgs-unstable",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "octopilot": {
+      "inputs": {
+        "flake-parts": [
+          "devenv-k8s",
+          "flake-parts"
+        ],
+        "nixpkgs": [
+          "devenv-k8s",
+          "nixpkgs"
+        ]
+      },
+      "locked": {
+        "lastModified": 1724261402,
+        "narHash": "sha256-ablZLlVPhrC5cShu2/5e80cbrR5MOMGc7Fpgf37HNbQ=",
+        "owner": "jashandeep-sohi",
+        "repo": "octopilot",
+        "rev": "b17d9033a57525c63fc973e3d1108fefdec62210",
+        "type": "github"
+      },
+      "original": {
+        "owner": "jashandeep-sohi",
+        "ref": "fix/commit-with-api",
+        "repo": "octopilot",
+        "type": "github"
+      }
+    },
+    "root": {
+      "inputs": {
+        "devenv-k8s": "devenv-k8s",
+        "devenv-root": "devenv-root_2",
+        "flake-parts": [
+          "devenv-k8s",
+          "flake-parts"
+        ],
+        "nixpkgs": [
+          "devenv-k8s",
+          "nixpkgs"
+        ]
+      }
+    },
+    "skaffold": {
+      "inputs": {
+        "flake-parts": [
+          "devenv-k8s",
+          "flake-parts"
+        ],
+        "gomod2nix": "gomod2nix",
+        "nixpkgs": [
+          "devenv-k8s",
+          "nixpkgs"
+        ]
+      },
+      "locked": {
+        "lastModified": 1711736775,
+        "narHash": "sha256-wRzy040KH+IfgK+g3Dj4Re4hb1S+G3xaxJ41TrsPEvE=",
+        "owner": "jashandeep-sohi",
```

Comment:
Why do we use your personal iteration of skaffold?

Reply:
I think it would make me feel marginally better to put the fixes onto an LCOGT org fork.